tools/vagrant/custom/README.md

```diff
@@ -114,7 +113,7 @@ vagrant ssh -- ip route get 8.8.8.8 | awk '{print $NF; exit}'
 From your _host_, configure `wsk` to use your Vagrant-hosted OpenWhisk deployment and run the "echo" action again to test.
 ```
 # Set your OpenWhisk Namespace and Authorization Key.
-  wsk property set --apihost `vagrant ssh -- ip route get 8.8.8.8 | awk '{print $NF; exit}'` --namespace guest --auth `vagrant ssh -- cat openwhisk/config/keys/auth.guest`
+  wsk property set --apihost `vagrant ssh -- ip route get 8.8.8.8 | awk '{print $NF; exit}'` --namespace guest --auth `vagrant ssh -- cat ansible/files/auth.guest`
```

Comment:
I'm assuming when we remove ant from doing this work, we delete all the keys from `config/keys/` for now we have to make sure that are duplicate files.